molecule/centos/molecule.yml

```diff
@@ -17,6 +18,8 @@ platforms:
       - /sys/fs/cgroup:/sys/fs/cgroup:ro
 provisioner:
   name: ansible
+  env:
+    PSQL_VERSION: ${PSQL_VERSION:-9.6}
```

Comment:
This uses the environment variable, but defaults to "9.6" if it is not set (e.g. for a local `molecule test`).